class.jetpack-gutenberg.php

```diff
@@ -458,7 +458,7 @@ public static function get_availability() {
 		$available_extensions = array();
 
 		foreach ( self::$extensions as $extension ) {
-			$is_available = self::is_registered( 'jetpack/' . $extension ) ||
+			$is_available = self::is_registered( 'jetpack/' . $extension ) &&
```

Comment:
I was thinking about this and I'm not sure, but I think the logic should be "if it's registered and there's no entry in `availability`, or there's an entry in `avaliability` and it's `true`, so this line would be something like:

```
$is_available = ( self::is_registered( 'jetpack/' . $extension ) && ! isset( self::$availability[ $extension ] ) ||
```

Reply:
I don't think this is right. When a block is set as unavailable we keep a record in the array for example for OpenTable we set the availability to this:

```Array
(
    [reason] => missing_plan
    [details] => Array
        (
            [required_feature] => opentable
            [required_plan] => value_bundle
        )

)```

Reply:
Oh wait I just read this again, you're right, sorry